ub_object_client/client.py

```diff
@@ -1,69 +1,73 @@
+        self.balance = to_money(self._data['balance'])
+        self.created = to_datetime(self._data['created'])
+        self.prebill_days = int(self._data['prebill_days'])
+        self.acct_balance = to_money(self._data['acct_balance'])
+        self.acctmgr = int(self._data['acctmgr'])
+        self.active = int(self._data['active'])
+        self.address = str(self._data['address'])
+        self.auto_apply_credit = bool(self._data['auto_apply_credit'])
+        self.business = int(self._data['business'])
+        self.charge_days = int(self._data['charge_days'])
+        self.checkname = str(self._data['checkname'])
+        self.city = str(self._data['city'])
+        self.class_id = int(self._data['class_id'])
+        self.comments = str(self._data['comments'])
+        self.commission = to_money(self._data['commission'])
+        self.commission_rate = to_money(self._data['commission_rate'])
+        self.commission_type = int(self._data['commission_type'])
+        self.company = str(self._data['company'])
+        self.country = str(self._data['country'])
+        self.credit_balance = to_money(self._data['credit_balance'])
+        self.credit_bool = bool(self._data['credit_bool'])
+        self.crv = int(self._data['crv'])
+        self.datedue = bool(self._data['datedue'])
+        self.discount = to_money(self._data['discount'])
+        self.discount_type = int(self._data['discount_type'])
+        self.email = str(self._data['email'])
+        self.fax = str(self._data['fax'])
+        self.full_name = str(self._data['full_name'])
+        self.grace_due = int(self._data['grace_due'])
+        self.inv_balance = to_money(self._data['inv_balance'])
+        self.invoice_delivery = bool(self._data['invoice_delivery'])
+        self.last = str(self._data['last'])
+        self.late_fee_scheme_id = int(self._data['late_fee_scheme_id'])
+        self.latest_inv = to_datetime(self._data['latest_inv'])
+        self.listed_company = str(self._data['listed_company'])
+        self.login = str(self._data['login'])
+        self.managed = str(self._data['managed'])
+        self.password_timeout = int(self._data['password_timeout'])
+        self.password_changed = to_datetime(self._data['password_changed'])
+        self.phone = to_datetime(self._data['phone'])
+        self.prebill_method = int(self._data['prebill_method'])
+        self.prefer_lang = int(self._data['prefer_lang'])
+        self.priority = int(self._data['priority'])
+        self.prorate_min_days = int(self._data['prorate_min_days'])
+        self.qblistid = int_or_none(self._data['qblistid'])
+        self.retry_every = int(self._data['retry_every'])
+        self.state = str(self._data['state'])
+        self.tier_commission = to_money(self._data['tier_commission'])
+        self.tier_commission_rate = to_money(self._data['tier_commission_rate'])
+        self.tier_commission_type = int(self._data['tier_commission_type'])
+        self.zip = str(self._data['zip'])
```

Comment:
for now I have decided to not consider nonexisting value. This will be adressed in a separate review.